site/zh-cn/guides/thread-primer/index.md

```diff
@@ -22,11 +22,11 @@ Thread 的主要特性包括：
 * 网络及路由概念
 * IPv6
 
-本入门教程基于 Thread Specification V1.1.1。Thread Specification 可以在 [threadgroup.org](http://threadgroup.org/ThreadSpec) 中获取。
+本入门教程基于 Thread Specification V1.2。Thread Specification 可以在 [threadgroup.org](https://www.threadgroup.org/support#specifications) 中获取。
```

Comment:
I think we'd remove the first sentence to align with the English version. @librasungirl can you confirm?

Reply:
Actually based on the earlier comments, we can probably just remove "V1.2" and we'll be good.  Rongli should confirm though.

Reply:
Go ahead and just remove the version number of "V1.2" only here

Reply:
OK, np. Done.